core/client/src/main/java/alluxio/client/RemoteBlockReader.java

```diff
@@ -40,13 +35,7 @@ private Factory() {} // prevent instantiation
      * @return a new instance of {@link RemoteBlockReader}
      */
     public static RemoteBlockReader create() {
-      try {
-        return CommonUtils.createNewClassInstance(
-            Configuration.<RemoteBlockReader>getClass(PropertyKey.USER_BLOCK_REMOTE_READER_CLASS),
```

Comment:
do we depreciate `USER_BLOCK_REMOTE_READER_CLASS`? If yes, could you comment in `PropertyKey.java` for this property?